kubernetes/couchdb/docker/init.sh

```diff
@@ -18,6 +24,30 @@ pushd /openwhisk
     echo "$AUTH_WHISK_SYSTEM" > /openwhisk/ansible/files/auth.guest
   fi
 
+  # start couchdb with a background process
+  couchdb -b
+
+  # wait for couchdb to be up and running
+  TIMEOUT=0
+  echo "wait for CouchDB to be up and running"
+  until [ $TIMEOUT -eq 25 ]; do
+    echo "waiting for CouchDB to be available"
+
+    if [ -n $(/etc/init.d/couchdb status | grep 'running') ]; then
+      echo "CouchDB is up and running"
+      break
+    fi
+
+    sleep 0.2
+    let TIMEOUT=TIMEOUT+1
+  done
+
+  if [ $TIMEOUT -eq 25 ]; then
+    echo "failed to setup CouchDB"
+    exit 1
+  fi
+
+
   # setup and initialize DB
   pushd ansible
     ansible-playbook -i environments/local setup.yml \
```

Comment:
yes. I wanted to make sure it always pulls the latest OW master code. That way I don't need to keep updating the image. Of course, this also means that if it fails to deploy then the image will need to be fixed as a whole